enhancements/update/phased-rollouts.md

```diff
@@ -0,0 +1,242 @@
+    * `phasedRollouts` (optional [array][json-array]) of rules for calculating rollout duration.
+        When `phasedRollouts` is empty or does not contain any matching rules, the default duration is `P0S` (i.e. edges appear immediately at the `start` time).
+        When there are multiple matching rules, consumers MUST apply only the `duration` set by the highest-precedence matching rule.
+        Each matching criterion MUST NOT appear more than once in the `phasedRollouts` array; for example, it is invalid to have:
+
+        ```yaml
+        channels:
+        - name: invalid-example
+          phasedRollouts:
+          - fromVersion: patch
+            duration: P1D
+          - fromVersion: patch
+            duration: P2D
+        ```
+
+        Each entry contains the following properties:
+        * `duration` (required [string][json-string], an [RFC 3339 `duration`][rfc-3339-p13]) sets the duration of the phased rollout.
+
+            There MAY be a single entry with no other properies besides `duration`.
+            When such an entry exists, consumers MUST treat it as the new default duration.
+            In other words, it matches all edges, but at the lowest precedence.
+        * `fromVersion` (optional [string][json-string]) describing the edges that match this rule, with the following possiblities:
+
+            * `minor`, for edges where the source and target differ by [minor version][semver-minor].
+            * `patch`, for edges where the source and target differ by [patch versions][semver-patch].
+
+            Other values may be possible in future versions, and consumers MUST treat unrecognized `fromVersion` values as acceptable, but non-matching rules.
+
+* `versions` (optional [array][json-array]) of releases which belong to the channel.
+    Each entry contains the following properties:
+    * `name` (required [string][json-string]) with the release name.
+    * `start` (required [string][json-string], an [RFC 3339 `iso-date-time`][rfc-3339-p13]) with the rollout start time.
+        Consumers MUST promote the named release to the configured `channels` immediately at `start`, but MUST NOT promote the named release to the configured `channels` before `start`.
+        Consumers MUST promote edges between promoted releases at or after `start` and at or before `start` + `duration`, but MAY choose any time they wish during that interval.
+
+[The schema version][graph-data-schema-version] would also be bumped to 2.0.0, because this is a backwards-incompatible change.
+
+This is the smallest pivot from the current schema, but provides no mechanism for more detailed control.
+Choosing this approach and later pivoting to [an alternative](#alternatives) later would be possible, but increases the development cost and possible support commitments.
+However, this approach does allow for additional matching rules to be added later under `phasedRollout` without breaking backwards compatibility.
```

Comment:
This proposal should contain the workflow when we find issues during the phased rollout. For example if we started the phased rollout but we found a blocker issue and want to stop the roll out. Also how to pause the phased rollout.

Reply:
> This proposal should contain the workflow when we find issues during the phased rollout.

9b1cb55 -> eab2408 adds a section walking through this, where I try to explain why I don't think we need a stop/pause button.